CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+# dbt_shopify v0.17.0
+
+[PR #100](https://github.com/fivetran/dbt_shopify/pull/100) includes the following updates:  
+
+## Breaking Changes  
+- In the source package, introduced the `inventory_quantity` source and the `stg_shopify__inventory_quantity` model to support downstream quantity tracking. 
```

Comment:
I'd maybe add a comment about how `quantity` was deprecated from `inventory_level`, just so it's easy to track why this change was needed

Reply:
Added